charts/kusion/README.md.gotmpl

```diff
@@ -16,7 +16,13 @@
 
 ## Installing the Chart
 
-> Note: A valid kubeconfig configuration is required for Kusion to function properly. You must either use the installation script, provide your own kubeconfig in values.yaml, or set it through the --set parameter
+> **Important:** Kusion requires a valid kubeconfig configuration to function properly. You can provide it via:
+> - Installation script (recommended)
```

Comment:
The default kubeconfig path should be explicitly documented as a security consideration, as it's a common location for sensitive credentials.